packages/database/src/core/util/libs/parser.ts

```diff
@@ -122,12 +124,14 @@ export const parseURL = function(
   secure: boolean;
   scheme: string;
   pathString: string;
+  namespace: string;
```

Comment:
Do we still need `subdomain`?  (seems like `namespace` should probably replace it)

Reply:
I'm not sure. `subdomain` means something different than `namespace` and we are relying on that distinction to determine whether we need to pass query parameters in the `RepoInfo` class.